# Задача № 1

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].

Найдите ее среднее значение и дисперсию.

Среднее значение - это по сути мат ожидание: M(X) = (A+B)/2, дисперсия: ((B-A) ** 2) / 12

In [1]:
m_x = (800 + 200) / 2
d_x = ((800 - 200) ** 2) / 12
m_x, d_x

(500.0, 30000.0)

Ответ: M(X) = 500, D(X) = 30000

# Задача № 2

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.

Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?

Если да, найдите ее.

In [4]:
b = (12*0.2) ** 1/2 + 0.5
m_x = (b + 0.5) / 2
b, m_x

(1.7000000000000002, 1.1)

Ответ: M(x) = 1.1, b = 1.7

# Задача № 3

Непрерывная случайная величина X распределена нормально и задана плотностью распределения

f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32).

Найдите:

а). M(X)

б). D(X)

в). std(X) (среднее квадратичное отклонение)

из условия видно, что сигма = 4 (пункт в)), отсюда D(X) = 16 (пункт б)), М(X) = -2 (пункт а)) - все данные берем из уравнения плотности распределения

Ответ: а) -2, б) 16, в) 4

# Задача № 4

Рост взрослого населения города X имеет нормальное распределение.

Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.

Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а). больше 182 см

б). больше 190 см

в). от 166 см до 190 см

г). от 166 см до 182 см

д). от 158 см до 190 см

е). не выше 150 см или не ниже 190 см

ё). не выше 150 см или не ниже 198 см

ж). ниже 166 см.

Решение:

запишем вводные: M(X) = 174, sigma = 8, D(X) = 64

In [27]:
import pandas as pd
import numpy as np
from scipy import stats

In [28]:
loc = 174
scale = 8

In [29]:
norm = stats.norm(loc, scale)
norm

In [30]:
norm_1 = 1 - norm.cdf(182)
norm_1

0.15865525393145707

In [31]:
norm_2 = 1- norm.cdf(190)
norm_2

0.02275013194817921

In [33]:
norm_3 = norm.cdf(190) - norm.cdf(166)
norm_3

0.8185946141203637

In [34]:
norm_4 = norm.cdf(182) - norm.cdf(166)
norm_4

0.6826894921370859

In [35]:
norm_5 = norm.cdf(190) - norm.cdf(158)
norm_5

0.9544997361036416

In [37]:
norm_6 = norm.cdf(150) + (1 - norm.cdf(190))
norm_6

0.0241000299798093

In [38]:
norm_7 = norm.cdf(150) + (1 - norm.cdf(198))
norm_7

0.0026997960632601965

In [39]:
norm_8 = norm.cdf(166)
norm_8

0.15865525393145707

Ответ:

а). больше 182 см: 0.15865525393145707

б). больше 190 см: 0.02275013194817921

в). от 166 см до 190 см: 0.8185946141203637

г). от 166 см до 182 см: 0.6826894921370859

д). от 158 см до 190 см: 0.9544997361036416

е). не выше 150 см или не ниже 190 см: 0.0241000299798093

ё). не выше 150 см или не ниже 198 см: 0.0026997960632601965

ж). ниже 166 см.: 0.15865525393145707

# Задача № 5

На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от

математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

решение: сигма = 5, поэтому

In [45]:
loc = 178
scale = 5

In [46]:
number_s = (190 -loc) / scale
number_s

2.4

ответ: на 2.4 сигмы отклоняется рост человека равный 190